src/xpanse-api/generated/models/DeployVariable.ts

```diff
@@ -113,4 +113,4 @@ export class DeployVariable {
 }
 
 export type DeployVariableKindEnum = 'fix_env' | 'fix_variable' | 'env' | 'variable';
-export type DeployVariableTypeEnum = 'string' | 'number';
+export type DeployVariableTypeEnum = 'string' | 'number' | 'boolean';
```

Comment:
is this change from new API.. then we must check in the corresponding api.json also pls